src/runtime/storageNG/tests/handle-test.ts

```diff
@@ -73,6 +76,24 @@ async function getSingletonHandle(primitiveType: Type, particle?: MockParticle,
   return handle;
 }
 
+async function getEntityHandle(schema: Schema, muxId: string, particle?: MockParticle, canRead=true, canWrite=false):
+    Promise<EntityHandle<Entity>> {
+  const fakeParticle: Particle = (particle || new MockParticle()) as unknown as Particle;
+  const handle = new EntityHandle(
+    'me',
+    new StorageProxy(
+      'id',
+      new MockStore<CRDTEntityTypeRecord<Identified, Identified>>(),
+      new EntityType(schema),
+      null),
+    IdGenerator.newSession(),
+    fakeParticle,
+    canRead,
+    canWrite,
+    muxId) as unknown as EntityHandle<Entity>;
```

Comment:
you should be able to say new EntityHandle<Entity>(...) directly, rather than casting.

This is also really hard to read all spread out, consider consolidating onto a couple of lines.